tests/test_hep_bdFFT.py

```diff
@@ -49,106 +49,46 @@ def test_fft_from_FFT():
 
     expected = [
         {
-            'creation_datetime': '2016-04-01T15:14:38',
-            'filename': '0029558261904692',
-            'flags': [
-                'HIDDEN',
-            ],
-            'format': '.xml',
-            'path': '/opt/cds-invenio/var/data/files/g122/2457396/content.xml;1',
-            'type': 'Main',
-            'version': 1,
+            'key': '0029558261904692.xml',
+            'hidden': True,
+            'url': '/afs/cern.ch/project/inspire/PROD/var/data/files/g122/2457396/content.xml;1',
         },
     ]
     result = hep.do(create_record(snippet))
 
-    assert validate(result['_fft'], subschema) is None
-    assert expected == result['_fft']
-
-    expected = [
-        {
-            'a': '/opt/cds-invenio/var/data/files/g122/2457396/content.xml;1',
-            'f': '.xml',
-            'n': '0029558261904692',
-            'o': [
-                'HIDDEN',
-            ],
-            's': '2016-04-01 15:14:38',
-            't': 'Main',
-            'v': 1,
-        },
-    ]
-    result = hep2marc.do(result)
-
-    assert expected == result['FFT']
-
-
-def test_fft_from_FFT_percent_percent():
-    snippet = (
-        '<datafield tag="FFT" ind1="%" ind2="%">'
-        '  <subfield code="a">/opt/cds-invenio/var/tmp-shared/apsharvest_unzip_5dGfY5/articlebag-10-1103-PhysRevD-87-083514-apsxml/data/PhysRevD.87.083514/fulltext.xml</subfield>'
-        '  <subfield code="o">HIDDEN</subfield>'
-        '  <subfield code="t">APS</subfield>'
-        '</datafield>'
-    )  # record/1094156
+    assert validate(result['documents'], subschema) is None
+    assert expected == result['documents']
 
-    assert '_fft' not in hep.do(create_record(snippet))
 
-
-def test_fft_from_FFT_ignores_context():
```

Comment:
Adapt the test to the new code. There was a point to this test, if you delete it you lose the knowledge that is stored in it!

Reply:
the point is that you should ignore FFT tags whose `f` subfield ends in context.